src/main/java/com/github/ygimenez/method/Pages.java

```diff
@@ -1548,9 +1549,9 @@ public static void modifyButtons(Message msg, Map<String, Function<Button, Butto
 		List<ActionRow> rows = new ArrayList<>(msg.getActionRows());
 
 		for (ActionRow ar : rows) {
-			List<Component> row = ar.getComponents();
+			List<ActionComponent> row = ar.getComponents().stream().map(itemComponent -> (ActionComponent) itemComponent).collect(Collectors.toList());
 			for (int i = 0; i < row.size(); i++) {
-				Component c = row.get(i);
+				ActionComponent c = row.get(i);
```

Comment:
Would be better to refrain from using subclass types since the library might support Components other than Button futurely.